### ISSUE TITLE
3.0.3 no longer extracts anything

### ISSUE THREAD

**Issue by ThiefMaster** [lingui/js-lingui#808]
**Describe the bug**
After updating to 3.0.3 no strings are extracted anymore.
On 3.0.2 it still works fine.

output in 3.0.2:

```
┌─────────────┬─────────────┬─────────┐
│ Language    │ Total count │ Missing │
├─────────────┼─────────────┼─────────┤
│ en (source) │     114     │    -    │
│ es          │     114     │   104   │
└─────────────┴─────────────┴─────────┘
```

and in 3.0.3:

```
┌─────────────┬─────────────┬─────────┐
│ Language    │ Total count │ Missing │
├─────────────┼─────────────┼─────────┤
│ en (source) │      0      │    -    │
│ es          │     114     │   104   │
└─────────────┴─────────────┴─────────┘
```

in `--verbose` i don't see any files during "Extracting messages from source files…" in 3.0.3, while I do in 3.0.2

**To Reproduce**
I noticed it when updating lingui in https://github.com/indico/newdle.

**Expected behavior**
Strings being extracted

**Additional context**

- jsLingui version 3.0.3
- Babel version
```
client@0.1.0 /home/adrian/dev/newdle/newdle/client
├── @babel/core@7.11.6
└─┬ react-scripts@3.4.3
  ├── @babel/core@7.9.0
  ├─┬ @svgr/webpack@4.3.3
  │ ├── @babel/core@7.11.6  deduped
  │ └─┬ @svgr/plugin-jsx@4.3.3
  │   └── @babel/core@7.11.6  deduped
  ├─┬ babel-jest@24.9.0
  │ └─┬ @jest/transform@24.9.0
  │   └── @babel/core@7.11.6  deduped
  ├─┬ babel-preset-react-app@9.1.2
  │ └── @babel/core@7.9.0
  └─┬ jest@24.9.0
    └─┬ jest-cli@24.9.0
      └─┬ jest-config@24.9.0
        └── @babel/core@7.11.6  deduped
```

package.json lingui section:

```json
  "lingui": {
    "catalogs": [
      {
        "path": "<rootDir>/src/locales/{locale}/messages",
        "include": [
          "<rootDir>/src"
        ],
        "exclude": [
          "**/node_modules/**"
        ]
      }
    ],
    "format": "po",
    "sourceLocale": "en",
    "locales": [
      "en",
      "es"
    ],
    "compileNamespace": "es"
  },
```

**Comment by ThiefMaster** [comment]
Looks like the `include` path needs a trailing slash now.. adding it fixed it, but I don't think this change in behavior was intended...

```diff
@@ -60,7 +60,7 @@
       {
         "path": "<rootDir>/src/locales/{locale}/messages",
         "include": [
-          "<rootDir>/src"
+          "<rootDir>/src/"
         ],
         "exclude": [
           "**/node_modules/**"
```

**Comment by tricoder42** [comment]
Yes, you're right. I've fixed it yesterday. The glob library needs to distinguish directories and files... I guess we could improve it by detecting folders automatically, but right now it needs to be set explicitly.

**Comment by tricoder42** [comment]
Closed in favor of #809
